platforms/java/com/area9innovation/flow/HttpSupport.java

```diff
@@ -213,12 +204,30 @@ public final Object httpCustomRequestWithTimeoutNative(String url, String method
 				responseHeaders.add(kv);
 			}
 
+			InputStream inputStream = null;
+			StringBuilder response = new StringBuilder();
+
+			if (responseCode == 200) {
```

Comment:
I would do: 
InputStream inputStream = con.getInputStream();
if (inputStream == null) {
   inputStream = con.getErrorStream();
}

Reply:
200 might not be the only none error response.